hathor/manager.py

```diff
@@ -492,19 +487,17 @@ def _initialize_components(self) -> None:
                 sys.exit()
 
         # restart all validations possible
-        deps_size = self.tx_storage.count_deps_index()
-        if deps_size > 0:
-            self.log.debug('run pending validations', deps_size=deps_size)
+        if self.tx_storage.indexes.deps.has_needed_tx():
+            self.log.debug('run pending validations')
```

Comment:
run or running?

Reply:
To me it makes more sense to say `do X` when we decide to do X and before actually doing X, and say `doing X` when we're in the middle of doing X (and possible log `doing X` multiple times). But we don't have a consistent logging style, so I don't mind either way.

Reply:
Ok. Thanks!